src/templates/blog-post-overview.tsx

```diff
@@ -82,6 +85,19 @@ export const BlogPageQuery = graphql`
       }
     }
 
+    blogHeader: contentfulSectionHeader(
+      slug: { eq: "blog-introduction" }
+      node_locale: { eq: $language }
+    ) {
+      kicker
+      headline
+      paragraphs {
+        paragraph {
+          paragraph
+        }
+      }
+    }
+
```

Comment:
@brataaaan 

I don't understand why you chose to author the content model like this: A section header mode + a separate section paragraph. Because we have theoretically multiple paragraphs? It would be much simpler to have the paragraph integrated in the section header model and saves a content model.

A similar thing happened with the leadbox content model (leadbox + link + contact)

- [ ] Please explain the content model. Thank you.

![CleanShot 2023-08-29 at 09 42 29](https://github.com/satellytes/satellytes.com/assets/1701755/3d1e4aef-61cf-4d52-9324-b56050eda39f)